extensions/wikidata/module/scripts/dialogs/manage-account-dialog.js

```diff
@@ -116,7 +116,7 @@ ManageAccountDialog.displayPasswordLogin = function (onSuccess) {
     ManageAccountDialog.displayOwnerOnlyConsumerLogin(onSuccess);
   });
 
-  elmts.loginForm.submit(function (e) {
+  elmts.loginButton.click(function (e) {
```

Comment:
I think it is preferable to bind to the `<form>`'s submit event rather than the login button click event, for accessibility reasons. For instance, if you press "Enter" in one of the input fields, it will trigger a form submit event, but not a button click. So I would rather keep the event binding as it was, and make the `<form>` encompass the buttons in their new location.

Reply:
Changed the event back to submit.